configs/katello_37.yaml

```diff
@@ -0,0 +1,207 @@
+      katello-3.7-rhel7-override:
+        katello-3.7-rhel7: 0
+      katello-3.7-rhel7:
+        katello-3.7-thirdparty-rhel7: 0
+      katello-3.7-client-rhel7:
+        katello-3.7-rhel7: 0
+        katello-3.7-thirdparty-pulp-rhel7: 2
+    external_repos:
+      - rhel-7.0-server
+      - rhel-7.0-server-optional
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+      - tfm-ror51-rhel-7
+      - centos-7-server
+  - name: katello-3.7-rhel6
+    based_off: katello-3.7-nightly-rhel6
+    helper_tags:
+      katello-3.7-thirdparty-rhel6: katello-thirdparty-rhel6
+      katello-3.7-rhel6-override: null
+      katello-3.7-client-rhel6: null
+    build_target: katello-3.7-rhel6-build
+    build_package_group_source_tag: katello-nightly-rhel6-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.7-rhel6-build:
+        katello-3.7-rhel6-override: 0
+      katello-3.7-rhel6-override:
+        katello-3.7-rhel6: 1
+      katello-3.7-rhel6:
+        katello-3.7-thirdparty-rhel6: 0
+      katello-3.7-client-rhel6:
+        katello-3.7-rhel6: 0
+        katello-3.7-thirdparty-pulp-rhel6: 2
+    external_repos:
+      - rhel-6.6-server
+      - rhel-6.6-server-optional
+      - epel-6
+  - name: katello-3.7-rhel5
+    based_off: katello-nightly-rhel6
+    helper_tags:
+      katello-3.7-rhel5-override: null
+      katello-3.7-client-rhel5: null
+    build_target: katello-3.7-rhel5-build
+    build_package_group_source_tag: katello-nightly-rhel5-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.7-rhel5-build:
+        katello-3.7-rhel5-override: 0
+      katello-3.7-rhel5-override:
+        katello-3.7-rhel5: 0
+      katello-3.7-rhel5: {}
+      katello-3.7-client-rhel5:
+        katello-3.7-rhel5: 0
+        katello-3.7-thirdparty-pulp-rhel5: 2
+    external_repos:
+      - rhel-5.9-server
+      - epel-5
+  - name: katello-3.7-fedora26
+    based_off: katello-nightly-fedora26
+    helper_tags:
+      katello-3.7-client-fedora26: null
+    build_target: katello-3.7-fedora26-build
+    build_package_group_source_tag: katello-nightly-fedora26-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.7-fedora26-build:
+        katello-3.7-fedora26: 0
+      katello-3.7-fedora26: {}
+      katello-3.7-client-fedora26:
+        katello-3.7-fedora26: 0
+    external_repos:
+      - fedora-26-updates
+      - fedora-26
+  - name: katello-3.7-fedora27
+    based_off: katello-nightly-fedora27
+    helper_tags:
+      katello-3.7-client-fedora27: null
+    build_target: katello-3.7-fedora27-build
+    build_package_group_source_tag: katello-nightly-fedora27-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.7-fedora27-build:
+        katello-3.7-fedora27: 0
+      katello-3.7-fedora27: {}
+      katello-3.7-client-fedora27:
+        katello-3.7-fedora27: 0
+    external_repos:
+      - fedora-27-updates
+      - fedora-27
+  - name: katello-nightly-sles12
```

Comment:
You'll want to update these two sles tags for to include version, such as:   
name: katello-3.7-sles12
base_off: katello-nightly-sles12